src/index.html

```diff
@@ -12,5 +12,28 @@
 </head>
 <body>
   <app-root></app-root>
+  <!-- Firebase App (the core Firebase SDK) is always required and must be listed first -->
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-app.js"></script>
+
+  <!-- Add Firebase products that you want to use -->
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-auth.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-firestore.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-database.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.4/firebase-storage.js"></script>
+
+  <script>
+    // Your web app's Firebase configuration
+    var firebaseConfig = {
```

Comment:
Can we get this config somewhere safe?

Reply:
@thepushkarp we should angularfire for firebase instade of firebase.js

and for that config make one ts file and put it there and add it to ignore

Reply:
Sure @aashutoshrathi @parth181195